.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,773 @@
+
+    # Fix arguments for methods
+    for keyword in method_keywords:
+        if keyword in arguments:
+            arguments.remove(keyword)
+
+    # Return
+    result = Function(name=name, arguments=arguments)
+    return result
+
+
+def extract_docstring(func_name, line_number, lines):
```

Ruff (0.12.2)

`extract_docstring` is too complex (12 > 10)

(C901)